examples/with-tailwind/packages/ui/tailwind.config.ts

```diff
@@ -1,5 +1,5 @@
 import type { Config } from "tailwindcss";
-import sharedConfig from "@repo/tailwind-config";
+import sharedConfig from "../tailwind-config/tailwind.config";
```

Comment:
boundaries error!

Reply:
Betrayed by my editor auto-updating paths! Egad!

Reply:
If only we had a feature for this .... 😄